gpytorch/variational/_variational_strategy.py

```diff
@@ -97,6 +122,113 @@ def kl_divergence(self):
             kl_divergence = torch.distributions.kl.kl_divergence(self.variational_distribution, self.prior_distribution)
         return kl_divergence
 
+    @cached(name="inducing_model")
+    def inducing_model(self):
+        with torch.no_grad():
+            inducing_noise_covar, inducing_mean = self.pseudo_points
```

Comment:
like my other comment, I would be careful about overloading term 'inducing', really you're converting/transforming the variational GP into an equivalent ExactGP. Something like `exact_transform` may be better. Make sure this cache is getting cleared whenever the underlying variational GP changes.

Reply:
Went with `amortized_exact_gp` which should be clear the type of perspective we're going for.